src/stories/SectionTeaserCard.stories.js

```diff
@@ -52,7 +52,7 @@ const mock = [
     image: API.image,
     to: '/visit/foo/barf/',
     category: 'Curabitur ',
-    title: 'Vel Quam Elementum',
```

Comment:
I am curious why we need this change? Do longer strings break new styles?

Reply:
I was just checking that this was working.